clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ConsumerAvroExample.java

```diff
@@ -46,10 +46,10 @@ public static void main(final String[] args) throws Exception {
     //   ssl.endpoint.identification.algorithm=https
     //   sasl.mechanism=PLAIN
     //   bootstrap.servers=<CLUSTER_BOOTSTRAP_SERVER>
-    //   sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<CLUSTER_API_KEY>" password="<CLUSTER_API_SECRET>";
+    //   sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
```

Comment:
Can we remove all this and replace it with a link to the new README?

e.g.

```
    // Load properties from a local configuration file
    // Create the configuration file (e.g. at '$HOME/.confluent/librdkafka.config') with configuration parameters
    // to connect to your Kafka cluster, which can be on your local host, Confluent Cloud, or any other cluster.
    // Follow these detailed instructions to properly create this file: https://github.com/confluentinc/configuration-templates/tree/master/README.md
```

Should be done for all instances, including:

java/src/main/java/io/confluent/examples/clients/cloud/StreamsExample.java:        // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/ConsumerAvroExample.java:    // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/ProducerAvroExample.java:    // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/StreamsAvroExample.java:        // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/ConsumerExample.java:    // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/ConsumerExamplePageviews.java:    // The configuration properties defined in the configuration file are assumed to include:
java/src/main/java/io/confluent/examples/clients/cloud/ProducerExample.java:    // The configuration properties defined in the configuration file are assumed to include: